papaya-web/src/main/java/pl/edu/agh/papaya/rest/sprints/controller/SprintsController.java

```diff
@@ -0,0 +1,73 @@
+package pl.edu.agh.papaya.rest.sprints.controller;
+
+import com.google.common.collect.Lists;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import javax.validation.Valid;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import pl.edu.agh.papaya.api.model.SprintCreationRequest;
+import pl.edu.agh.papaya.api.model.SprintDto;
+import pl.edu.agh.papaya.api.model.SprintStateDto;
+import pl.edu.agh.papaya.mappers.LocalDateTimePeriodMapper;
+import pl.edu.agh.papaya.mappers.SprintMapper;
+import pl.edu.agh.papaya.mappers.SprintStateMapper;
+import pl.edu.agh.papaya.model.LocalDateTimePeriod;
+import pl.edu.agh.papaya.model.Project;
+import pl.edu.agh.papaya.model.Sprint;
+import pl.edu.agh.papaya.model.SprintState;
+import pl.edu.agh.papaya.service.sprint.SprintService;
+
+public class SprintsController {
+
+    private static final List<SprintStateDto> ALL_SPRINT_STATE_DTOS = Lists.newArrayList(SprintStateDto.values());
+
+    @Autowired
```

Comment:
Na pierwszy rzut oka wydaje mi się, że to nie zadziała bo `SprintsController` nie ma możliwości bycia zarządzanym przez kontener. Nie jest tak? :open_mouth: